Archon.Http.Tests/FakeHttpHandler.cs

```diff
@@ -3,21 +3,17 @@
 using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
-using System.Web.Http;
-using System.Web.Http.Hosting;
 
-namespace Archon.WebApi
+namespace Archon.Http.Tests
 {
-	public class FakeHttpHandler : HttpMessageHandler
+	class FakeHttpHandler : HttpMessageHandler
```

Comment:
Actually, thinking about this further, we probably are going to need to maintain a 1.x version branch for legacy projects anyway. In v2, we do want to remove this publicly.

I'd like to get this merged though, so I can use this in the [releases project](https://github.com/civicsource/releases) to gather github stats.